crates/store/re_remote_store_types/src/v0/rerun.remote_store.v0.rs

```diff
@@ -355,6 +355,24 @@ pub struct RecordingInfo {
     #[prost(enumeration = "RecordingType", tag = "5")]
     pub typ: i32,
 }
+#[derive(Clone, PartialEq, ::prost::Message)]
```

Comment:
We should mark these files as `linguist-generated=true` in `.gitattributes` so they don't show up in diffs